code/modules/research/rdconsole.dm

```diff
@@ -153,7 +153,7 @@ doesn't have toxins access.
 	if(stored_research.research_points >= price)
 		investigate_log("[key_name(user)] researched [id]([price]) on techweb id [stored_research.id].", INVESTIGATE_RESEARCH)
 		if(stored_research == SSresearch.science_tech)
-			SSblackbox.record_feedback("associative", "science_techweb_unlock", 1, list("id" = "[id]", "price" = "[price]", "time" = "[SQLtime()]"))
+			SSblackbox.record_feedback("associative", "science_techweb_unlock", 1, list("id" = "[TN.name]", "price" = "[price]", "time" = "[SQLtime()]"))

```

Comment:
I meant to add name as another argument not to replace the id =P
Also the var you want is `TN.display_name`

Reply:
"Oh you can webedit something so simple!" - Me, to me